app/main/routes.py

```diff
@@ -133,6 +133,11 @@ def browse():
     elif series_id:
         browse_type = "series"
         browse_parameters["series_id"] = series_id
+        # sorting_orders["last_record_transferred"] = "asc"  # oldest first
```

Comment:
so for series we just sort by last record transferred date and num records held. Cool